schedule.json

```diff
@@ -1,4 +1,8 @@
 {
+  "v0.8": {
+    "start": "2012-06-25",
+    "end": "2013-03-11"
```

Comment:
https://nodejs.org/dist/v0.8.28/ was the last v0.8.x release in 2014: https://nodejs.org/en/blog/release/v0.8.28/